qiita_db/metadata_template/prep_template.py

```diff
@@ -337,6 +337,34 @@ def can_be_updated(self, columns):
 
             return False
 
+    def can_be_extended(self, new_samples, new_columns):
+        """Gets if the template can be can_be_extended
```

Comment:
Can you change this to `Whether the template can be extended or not`, same thing for the implementation of this method in sample_template.py.